src/test/java/com/conveyal/datatools/manager/models/transform/NormalizeFieldTransformationTest.java

```diff
@@ -0,0 +1,106 @@
+    @ParameterizedTest
+    @MethodSource("createCapitalizationCases")
+    public void testConvertToTitleCase(String input, String expected) {
+        NormalizeFieldTransformation transform = NormalizeFieldTransformation.create("table", "field");
+        assertEquals(expected, transform.convertToTitleCase(input));
+    }
+
+    private static Stream<Arguments> createCapitalizationCases() {
+        return Stream.of(
+            Arguments.of(null, ""),
+            Arguments.of("LATHROP/MANTECA STATION", "Lathrop/Manteca Station"),
+            Arguments.of("12TH@WEST", "12th@West"),
+            Arguments.of("12TH+WEST", "12th+West"),
+            Arguments.of("12TH&WEST", "12th&West"),
+            Arguments.of("904 OCEANA BLVD-GOOD SHEPHERD SCHOOL", "904 Oceana Blvd-Good Shepherd School"),
+            // Capitalization exceptions from config (found on MTC Livermore Route 1)
+            Arguments.of("EAST DUBLIN BART STATION", "East Dublin BART Station"),
+            Arguments.of("DUBLIN & ARNOLD EB", "Dublin & Arnold EB")
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("createCapitalizationCasesWithOwnExceptions")
+    public void testConvertToTitleCaseWithOwnExceptions(String input, String expected) {
+        NormalizeFieldTransformation transform = NormalizeFieldTransformation.create(
+            "table", "field", "NE, SW, de", null);
+        assertEquals(expected, transform.convertToTitleCase(input));
+    }
+
+    private static Stream<Arguments> createCapitalizationCasesWithOwnExceptions() {
+        return Stream.of(
+            // Capitalization exceptions from instance (quadrant street names)
+            Arguments.of("10TH STREET NE", "10th Street NE"),
+            Arguments.of("10TH STREET SW", "10th Street SW"),
+            Arguments.of("PONCE DE LEON", "Ponce de Leon")
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("createSubstitutionCases")
+    public void testPerformSubstitutions(String input, String expected) {
+        NormalizeFieldTransformation transform = NormalizeFieldTransformation.create("table", "field");
+        assertEquals(expected, transform.performSubstitutions(input));
+    }
+
+    private static Stream<Arguments> createSubstitutionCases() {
+        return Stream.of(
+            // Replace "@"
+            Arguments.of("12TH@WEST", "12TH at WEST"),
+            Arguments.of("12TH  @   WEST", "12TH at WEST"),
+            // Replace "+"
+            Arguments.of("12TH+WEST", "12TH and WEST"),
+            Arguments.of("12TH  +   WEST", "12TH and WEST"),
+            // Replace "&"
+            Arguments.of("12TH&WEST", "12TH and WEST"),
+            Arguments.of("12TH  &   WEST", "12TH and WEST"),
+            // Replace contents in parentheses and surrounding whitespace.
+            Arguments.of("14th St & Broadway (12th St BART) ", "14th St and Broadway")
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("createSubstitutionCasesWithOwnExceptions")
```

Comment:
Maybe a better name for "OwnExceptions" in the various method names is "CustomExceptions"?

Reply:
Updated in https://github.com/ibi-group/datatools-server/pull/371/commits/b2dd61e73bfedc146412d3c3c9386693d5799609.